models.js

```diff
@@ -96,3 +96,17 @@ exports.updateVotes = (votes, article) => {
   })
 })
 }
+
+exports.removeComment = (comment) => {
+  return db
+  .query("SELECT * FROM comments WHERE comment_id = $1;", [comment])
+  .then((result) => {
+    if (!result.rows[0]) {
+      return Promise.reject({
+        status: 404,
+        msg: `Comment not found`,
+      });
+    }
+  return db.query(`DELETE FROM comments WHERE comment_id = $1`, [comment])
+})
```

Comment:
So you could avoid two queries to the database by using `returning *` on your delete query. This gives you an array of the deleted rows. If that is empty then you know there was no comment of that id.